applications/app/views/fragments/newGalleryBody.scala.html

```diff
@@ -1,115 +0,0 @@
-@(page: model.GalleryPage)(implicit request: RequestHeader)
```

Comment:
If you do a `git mv` on these files it would be able to keep track of any changes regardless of the file rename